File: 2order-frontend/src/model/Racun.ts
import Porudzbina from "./Porudzbina";
import Sto from "./Sto";

interface Racun {
    id: number;
    vreme: Date;
    listaPorudzbina: Porudzbina[];
    stoId?: number;
    sto?: Sto;
    iznos: string;
}

export default Racun;

File: 2Order/2Order.Domain/Model/Racun.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;
using System.Text.Json.Serialization;

namespace _2Order.Domain.Model
{
    public class Racun
    {
        public Racun()
        {
            this.ListaPorudzbina = new List<Porudzbina>();
        }

        public int Id { get; set; }
        public DateTime Vreme { get; set; }
        public IList<Porudzbina> ListaPorudzbina { get; set; }
        public int? StoId { get; set; }
        public Sto Sto { get; set; }
        public float Iznos { get; set; }
    }
}
